src/aspire/utils/preprocess.py

```diff
@@ -302,3 +303,34 @@ def vol2img(volume, rots, L=None, dtype=None):
         im = im * m_reshape(np.exp(2*np.pi*1j*(grid2d['x'] +grid2d['y'])/(2*lv)), (lv, lv, 1))
 
     return np.real(im)
+
+
+def fuzzy_mask(L, r0, risetime, origin=None):
+    """
+    Create a centered 1D to 3D object of radius r0
```

Comment:
this docstring is not really clear, nor is the code (to me), though I was able to suss out what it was doing with time.

Reply:
Done in commit 03944b1.